src/wagtailmedia/static/wagtailmedia/js/tabs.js

```diff
@@ -0,0 +1,365 @@
+/**
```

Comment:
@thibaudcolas any reason not to use the non-transpiled ES6 version given that we dropped IE11 support in 2.15?